tests/phpunit/src/ConfigDefinitionTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ExTeradata\Tests\Unit\src;
+
+use Keboola\ExTeradata\ConfigDefinition;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+
+class ConfigDefinitionTest extends TestCase
+{
+    public function testGetParametersDefinitionReturnsArrayNodeDefinition(): void
```

Comment:
To mi přijde jako divný test. Takový jakože "fake na code coverage" :D

Reply:
Jo, je takovy useless, jeste ho upravim.